docs/advanced/gsdumprunner.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "GS Dump Runner"
+summary: "How to use the GS Dump Runner"
+draft: false
+toc: true
+sidebar_position: 3
+---
+
+The GS dump runner is designed to automatically run and compare a large number of GS dumps and output any differences found with pictures.
+
+`pcsx2-gsrunner` does not build by default. You need to right click in the solution explorer in VS, and select Build.
+
+:::info
+The GS dump runner is only available for Windows at the moment.
+:::
+
+To use:
+
+:::tip
+You can add the `-parallel 4` as a parameter for it to run multiple dumps at once (recommended).
```

Comment:
Either "add the parameter `-parallel 4` for it to run" or 'add `-parallel 4` as a parameter for it to run".